### ISSUE TITLE
Unable to Reproduce Your Results - Could You Please Provide the Trained Model or Trained Parameters?

### ISSUE THREAD

**Issue by Garhoode** [sengine-research/SpaceLM#8]
Hi, thank you for your excellent work!

I have carefully followed your instructions and successfully reproduced each step of the code without any major issues. However, the final results I obtained are significantly worse than those reported in your repository.

I’m not sure what went wrong. To investigate, I’ve tried increasing the number of training epochs (Stage 1: from 4 to 40, Stage 2: from 10 to 100), but the performance still falls short, even the loss shows nearly 0!

Here is my train command, 
`CUDA_VISIBLE_DEVICES=0 python train.py  --dataset_dir preprocessed_data_scene_script --dataset_name Scannet --model_path ./Qwen2.5-0.5B-Instruct --exp_path ./exp --exp_name space_lm_model_qwen_llm_lr_1e-6_point_lr_1e-5_2025_06_16 --stage_1_epochs 40 --stage_2_epochs 100 --batch_size 1 --gradient_accumulation_steps 16 --learning_rate 5e-6 --save_per_epoch 2`

Could you kindly provide the pre-trained model or any additional training details that might help with reproducing your results?

Thanks in advance for your help!

![Image](https://github.com/user-attachments/assets/5c3f6473-137b-430d-97ac-552d987939cd)
![Image](https://github.com/user-attachments/assets/81f0e8f4-57f3-4049-aaeb-9fa78d3cd715)
![Image](https://github.com/user-attachments/assets/5656fbac-869c-4694-8493-805568591b26)
